packages/server/graphql/mutations/helpers/removeFromOrg.ts

```diff
@@ -57,6 +59,7 @@ const removeFromOrg = async (
     dataLoader.get('users').loadNonNull(userId)
   ])
   dataLoader.clearAll('organizationUsers')
+  const {tms} = user
```

Comment:
running `yarn pg:generate` should remove tms from the schema & will make finding things like this easier, at least i hope!

Reply:
I've updated the description. Although I'm removing `tms` from the `User` table, I'm still querying it in the dataloader. I guess a different solution could also have been to add a trigger in PG to just get rid of the update logic in code. For performance reasons, especially for socket connects/disconnects I did not want to double the roundtrips to the db.